pkgs/tools/package-management/lix/common.nix

```diff
@@ -102,32 +96,33 @@ stdenv.mkDerivation {
     ++ lib.optionals enableDocumentation [
       "man"
       "doc"
+      "devdoc"
     ];
 
   strictDeps = true;
 
   nativeBuildInputs =
     [
       pkg-config
-      bison
       flex
       jq
       meson
       ninja
       cmake
       python3
-      doxygen
 
       # Tests
       git
       mercurial
       jq
       lsof
     ]
-    ++ lib.optionals (enableDocumentation) [
+    ++ lib.optionals isLegacyParser [ bison ]
+    ++ lib.optionals enableDocumentation [
       (lib.getBin lowdown)
       mdbook
       mdbook-linkcheck
+      doxygen
```

Comment:
This should always be in `nativeBuildInputs` as it's always checked for by Meson

Reply:
nope, that's because of the internal-api-docs feature, which we should be able to disable.

Reply:
confirmed fixed.